ethsigner/src/main/java/tech/pegasys/ethsigner/jsonrpc/response/JsonRpcErrorResponse.java

```diff
@@ -23,7 +24,8 @@
   private final Object id;
   private final JsonRpcError error;
 
-  public JsonRpcErrorResponse(final Object id, final JsonRpcError error) {
+  public JsonRpcErrorResponse(
```

Comment:
Are you using this POJO to parse JSON? (I thought that a `@JsonCreator`annotation was needed), if not, then there's no need for the `@JsonProperty` annotations

Reply:
It is being used to parse error responses during integration testing.
Apparently if all params are flagged as @JsonParameter the @JsonCreator can be dropped (but appears this may be undocumented) https://stackoverflow.com/questions/40843860/is-use-of-jsoncreator-necessary-when-using-jsonproperty - should we add it?